src/main/java/io/zipcoder/interfaces/Instructor.java

```diff
@@ -0,0 +1,25 @@
+package io.zipcoder.interfaces;
+
+public class Instructor extends Person implements Teacher {
+
+    private double totalStudyTime = 0;
+
+    public Instructor(long id, String name) {
+        super(id, name);
+    }
+
+    public double teach(Learner learner, double numberOfHours) {
+
+        return totalStudyTime += numberOfHours;
+
+    }
+
+    public double lecture(Learner learners[], double numberOfHours) {
+
```

Comment:
what is the purpose of the `Learner[]` argument if it is never used in the method definition?
Revisit the instructions regarding this method